dali/test/python/test_dali_cpu_only.py

```diff
@@ -805,6 +969,38 @@ def test_stack_cpu():
     for _ in range(3):
         pipe.run()
 
+def test_batch_permute_cpu():
+    pipe = Pipeline(batch_size=batch_size, num_threads=3, device_id=None)
+    data = fn.external_source(source = get_data, layout = "HWC")
```

Comment:
Why not adding `with pipe:` in this one? I ask, because you add it in one of the test cases above.

Reply:
Done